src/test/java/be/seeseemelk/mockbukkit/inventory/meta/ItemMetaMockTest.java

```diff
@@ -392,6 +392,14 @@ void getLore_LoreChangedAfterSet_LoreNotChanged()
 		assertEquals("world", lore.get(1));
 	}
 
+	@Test
+	void setLocalizedName_NewName_NameSetExactly()
+	{
+		meta.setLocalizedName("Some name");
+		assertTrue(meta.hasLocalizedName());
+		assertEquals("Some name", meta.getLocalizedName());
+	}
```

Comment:
An additional unit test for the default behaviour would be nice.
In other words: assertFalse for `meta.hasLocalizedName()` and asserting that the localized name is null by default.

Reply:
fix [`398c354` (#268)](https://github.com/MockBukkit/MockBukkit/pull/268/commits/398c354401cee6db7337304bfaefc0076729c8a1)